packages/sources/alpine/package.json

```
@@ -0,0 +1,41 @@
+{
+  "name": "@chainlink/alpine-adapter",
+  "version": "0.0.1",
```

We might not even need a changeset, but let's start out on v1.0.0 here
```suggestion
  "version": "1.0.0",
```